static-assets/scripts/main.js

```diff
@@ -97,6 +98,35 @@
 
         }
 
+        function setDocTitle(globalMenuData, toState) {
+          let docTitle;
+
+          // check if state is a globalMenu Item
+          const globalMenuItem = globalMenuData.find(o => o.id === toState.name);
+
+          // if globalMenuItem exists => check if globalMenuMessages has the id, or use the label as docTitle
+          if (globalMenuItem) {
+            docTitle = globalMenuMessages[globalMenuItem.id]
+              ? `${ formatMessage(globalMenuMessages[globalMenuItem.id]) } - Crafter CMS`
+              : `${ formatMessage(globalMenuMessages[globalMenuItem.label]) } - Crafter CMS`;
```

Comment:
The label should not be re-translated 😱 (test things please. this probably throws as it is.)